index.html

```diff
@@ -5,12 +5,96 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&display=swap" rel="stylesheet">
+
+    <title>Adiba's Bakery</title>
+    <link rel="stylesheet" href="/style.css">
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <div class="container">
+        <header class="header-description">
+            <div class="logo"></div>
+            <div class="flex-nav">
+                <div class="nav-button">
+                    <button class="nav-toggle" id="navToggle">
+                        <i class="fas fa-bars"></i>
+                    </button>
+                </div>
+            </div>
+
+            <div class="navbar-description">
+                <nav class="nav-links" id="nav-links">
+                    <ul>
+                        <li class="nav-link"><a href="#"> HOME</a></li>
+                        <li class="nav-link"><a href="#"> CAKES </a></li>
+                        <li class="nav-link"><a href="#"> ORDERIN</a></li>
+                        <li class="nav-link"><a href="#"> LESSONS</a></li>
+                        <li class="nav-link"><a href="#"> ABOUT</a></li>
+                    </ul>
+                </nav>
+            </div>
+    <div class="h2">
+        <h2> The best cakes in town delevered to your door.
+        </h2>
+    </div>
+
+    </header>
+
+    <section class="first-section">
+        <h1> Welcome</h1>
+        <p>
+            At our bakery, we only use the highest of quality of ingredients to produce the best tasting sponges,
+            because we believe a cake must be as gorgeous on the outside as it is on the inside. We take huge pride
+            in our large range of flavours and fillings, from the traditional to the funky there is something for
+            everyone.
+        </p>
+    </section>
+    <section class="second-section">
+    </section>
+    <section class="third-section">
+        <article class="third-section-article1"> art1</article>
+        <article class="third-section-article2"> art2 </article>
+        <article class="third-section-article3"> art3 </article>
+        <article class="third-section-article4">art 4 </article>
+    </section>
+
+    <footer>
+
+        <section id="footer">
+            <p> Join us on
+                <br>
+            <div class="footer-button">
+                <button>
+                    <i class="fas fa-bars"></i>
+                </button>
+            </div>
+            </p>
+
+            <div class="icon" id="iconList">
+                <div class="circle">
+                    <i class="fab fa-facebook-f"></i>
```

Comment:
when using icons it is a good idea to add an `aria-label` tag as this will help those who use assistive technology (for example users who rely on screen readers), for example you could do `aria-label="facebook"`